src/harness/fourslash.ts

```diff
@@ -3720,6 +3740,10 @@ namespace FourSlashInterface {
             this.state.verifySymbolAtLocation(startRange, declarationRanges);
         }
 
+        public typeOfSymbolAtLocation(range: FourSlash.Range, symbol: ts.Symbol, expected: string) {

```

Comment:
can't we get this from the quick info? I don't know whether we need a separate method for comparing the stringified type of a symbol.